apis/src/api/v1/auth/get_token_handler.rs

```diff
@@ -40,7 +40,14 @@ async fn get_token_helper(
     pool: Data<DbPool>,
 ) -> Result<String> {
     let mut conn = get_conn(&pool).await?;
-    let user = User::find_by_email(&bot.email, &mut conn).await?;
+
+    let user_result = User::find_by_email(&bot.email, &mut conn).await?
```

Comment:
This won't work. If the bot cannot be found via email, because you used the botname the function will automatically bubble up the error returned because of the "?" operator at the end.
Compare how we do it for normal login here: https://github.com/hiveboardgame/hive/blob/main/apis/src/functions/auth/login.rs#L17-L27

Reply:
So the correct way to do it is to add `.map_err(ServerFnError::new);` after the await ?

Reply:
No, you don't want to bubble up the result via the "?" operator. And I think you are missing a ";" there as well. So:

```suggestion
    let user_result = User::find_by_email(&bot.email, &mut conn).await;
```

Reply:
Okay, I think I get it now. I committed your proposed change